checkov/common/bridgecrew/platform_integration.py

```diff
@@ -111,8 +111,8 @@ def clean(self) -> None:
         self.bc_source_version: str | None = None
         self.timestamp: str | None = None
         self.scan_reports: list[Report] = []
-        self.bc_api_url = normalize_bc_url(os.getenv('BC_API_URL', "https://www.bridgecrew.cloud"))
-        self.prisma_api_url = normalize_prisma_url(os.getenv("PRISMA_API_URL"))
+        self.bc_api_url = normalize_bc_url(os.getenv('BC_API_URL'))
```

Comment:
do we want to delete this? or just not the default anymore?

Reply:
I deleted only the default value of BC, the attribute is still there.   
Didn't want to remove it because we will probably need that to work with our dev environments.